govlearn-api/src/main/java/com/unimuenster/govlearnapi/course/entity/Course.java

```diff
@@ -36,8 +36,9 @@ public class Course {
     protected String instructor;
     protected Boolean certificate;
     protected Skilllevel skilllevel;
-    protected String duration;
+    protected int durationInMinutes;
```

Comment:
Ich musste die Dauer ändern, weil ein String als Zeiteinheit einfach nicht suchbar ist.

Reply:
kann mir vostellen, dass das im FrontEnd Probleme bereiten könnte, lässt sich da nicht im Service converten?

Reply:
Nein, mit einem String nach kleiner/größer als zu suchen, würde bedeuten, dass man alle möglichen Zeiten sucht: 1 Stunde 1.1 Stunden 1.2 Stunde...
Die duration an das front end wieder von Minuten in Stunden zu konvertieren hört sich aber machbar an.

Reply:
done